Added params with detail for module:status command (#7099)
magento/devdocs: src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.md

This command has no prerequisites.

## Module status {#instgde-cli-subcommands-status}

Use the following command to list enabled and disabled modules:

```bash
bin/magento module:status <module-name> [--enabled] [--disabled]
```

where

> *  `<module-name>` is the name of the module for which you need to check the status.

Reviewer: Move this to last.
```suggestion
*  `<module-name>`specifies which module to return the status of. You cannot specify multiple module names.
```

> *  `--enabled` to list all the enabled modules.

Reviewer: ```suggestion
*  `--enabled` lists all enabled modules.
```

> *  `--disabled` to list all the disabled modules.

Reviewer: ```suggestion
*  `--disabled` lists all disabled modules.
```
